articles/industrial-iot/industrial-iot-platform-versions.md

```diff
@@ -5,21 +5,25 @@ author: jehona-m
+# Azure Industrial IoT Platform v2.8 LTS
+
+We are pleased to announce the declaration of Long-Term Support (LTS) for version 2.8. While we continue to develop and release updates to our ongoing projects on GitHub, we now also offer a branch that will only get critical bug fixes and security updates starting in July 2021. Customers can rely upon a longer-term support lifecycle for these LTS builds, providing stability and assurance for the planning on longer time horizons our customers require. The LTS branch offers customers a guarantee that they will benefit from any necessary security or critical bug fixes with minimal impact to their deployments and module interactions.  At the same time, customers can access the latest updates in the [main branch](https://github.com/Azure/Industrial-IoT) to keep pace with the latest developments and fastest cycle time for product updates. 
 
 ## Version history 
 
 |Version      |Type                   |Date         |Highlights                             |
 |-------------|-----------------------|-------------|---------------------------------------|
 |2.5.4        |Stable                 |March 2020   |IoT Hub Direct Method Interface, control from cloud without any additional microservices (standalone mode), OPC UA Server Interface, uses OPC Foundation's OPC stack - [Release notes](https://github.com/Azure/Industrial-IoT/releases/tag/2.5.4)|
 |[2.7.206](https://github.com/Azure/Industrial-IoT/tree/release/2.7.206)      |Stable                 |January 2021 |Configuration through REST API (orchestrated mode), supports Samples telemetry format as well as PubSub - [Release notes](https://github.com/Azure/Industrial-IoT/releases/tag/2.7.206)|
-|[2.8.0](https://github.com/Azure/Industrial-IoT/tree/release/2.8)        |Long-term support (LTS)|July 2021    |IoT Edge update to 1.1 LTS, OPC stack logging and tracing for better OPC Publisher diagnostics, Security fixes|
+|[2.8](https://github.com/Azure/Industrial-IoT/tree/release/2.8)        |Long-term support (LTS)|July 2021    |IoT Edge update to 1.1 LTS, OPC stack logging and tracing for better OPC Publisher diagnostics, Security fixes|
```

Comment:
We should be linking to the tags instead of `release/2.8` branch as this branch will be updated to point to `2.8.1` version.

So let's change:
`https://github.com/Azure/Industrial-IoT/tree/release/2.8` -> `https://github.com/Azure/Industrial-IoT/tree/2.8.0`
`https://github.com/Azure/Industrial-IoT/tree/release/2.8.1` -> `https://github.com/Azure/Industrial-IoT/tree/2.8.1`

Reply:
Let's add a link to release notes similar to `2.5.4` and `2.7.206` versions.

Reply:
will add link to release notes of 2.8.1 once it's finalized.